requirements.txt

```diff
@@ -4,8 +4,8 @@
 six>=1.9.0 # MIT
 PyYAML>=3.10.0,<6 # MIT
 pbr>=1.8 # Apache-2.0
-stevedore>=1.17.1,<3.0.0;python_version=='3.5' # Apache-2.0
-stevedore>=1.17.1;python_version>='3.6' # Apache-2.0
+stevedore>=1.17.1,<2; python_version < '3.0'  # Apache-2.0
```

Comment:
We are getting the change from upstream instead of ours

Reply:
This is intended - as it turned out our change was rather a workaround for a limitation of a particular pip version... With a proper pip version we don't need this workaround.

Nonetheless, I think we should try to be fully aligned with upstream, if there are no major reasons to be otherwise.